region/maridia/inner-pink.json

```diff
@@ -1274,6 +1276,56 @@
                     "Even when suitless, it's possible to stand far enough that the acid attack doesn't spawn.",
                     "Botwoon still gets hit. So there's a safe way to take no damage."
                   ]
+                },
+                {
+                  "name": "Back-Side Botwoon Microwave",
+                  "notable": true,
+                  "requires": [
+                    "canFightEnemiesOffCamera",
+                    "h_canNavigateUnderwater",
+                    "canPixelPreciseTrick",
+                    "Charge",
+                    "Plasma",
+                    "XRayScope"
+                  ],
+                  "note": "Stand on the appropriate pixel for shooting diagonally through the wall and use the microwave trick to defeat Botwoon.",
+                  "devNote": "Head through wall microwave is reasonable but should expect to cost some hits."
+                },
+                {
+                  "name": "Back-Side Botwoon Magic Pixel",
+                  "notable": true,
+                  "requires": [
+                    "canFightEnemiesOffCamera",
+                    "h_canNavigateUnderwater",
+                    "canPixelPreciseTrick",
+                    {
+                      "enemyKill": {
+                        "enemies": [ [ "Reverse Botwoon" ] ],
+                        "explicitWeapons": [ "Charge+Plasma", "Charge+Ice+Spazer" ]
+                      }
+                    }
+                  ],
+                  "note": [
+                    "There are a few pixels near the dividing wall where a charge beam shot will pass right through.",
+                    "Using angle down the spot is about a half tile from the wall.",
+                    "Using angle up, it is where Samus' front toe is on the seam in the floor."
+                  ]
+                },
+                {
+                  "name": "Plasma Shield",
+                  "notable": false,
+                  "requires": [
+                    "canFightEnemiesOffCamera",
+                    "h_canNavigateUnderwater",
+                    {
+                      "enemyKill": {
+                        "enemies": [ [ "Reverse Botwoon" ] ],
+                        "explicitWeapons": [ "Plasma Shield" ]
+                      }
+                    }
+                  ],
+                  "note": "Stand near the dividing wall and unleash the Plasma Special Beam Attack!",
+                  "devNote": "6 power bombs expected."
                 }
```

Comment:
Does there need to be a strat here as well for the supers?

Reply:
I was going to leave out the super only strat for being not fun.